### PR TITLE
Ape 672

### PR DESCRIPTION
**Pull request by tigredonorte**
adding basic style that should be used on any gorila-pages and webcomponents (styles for html, body, etc)

### CODE REVIEW

File: scss/_global.scss [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+  padding: 0;
+}
+
+@media (min-width: 1200px) {
+  .router-container>dashboard {
+    padding: $base-padding;
+  }
+}
+
+.product-card {
```
Comment (oliveira-gust) [inline scss/_global.scss:148]: This seems a bit too specific to be here... what do you think?

File: scss/_global.scss [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    overflow: initial;
+  }
+
+  &:not(:last-child) {
+    margin-right: 20px;
+  }
+}
+
+/* stylelint-disable max-nesting-depth */
+.blotter-buttons {
```
Comment (oliveira-gust) [inline scss/_global.scss:161]: Same as above

File: scss/_global.scss [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+.text-default {
+  color: $gorila-primary;
+}
+
+.text-error {
+  color: $negative;
+}
+
+/* stylelint-disable max-nesting-depth */
+blotter-form {
```
Comment (oliveira-gust) [inline scss/_global.scss:241]: Same as above

File: scss/_global.scss [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  text-transform: capitalize;
+
+  background-color: $extra-dark-gray;
+  border-radius: 5px;
+  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.07);
+}
+/* stylelint-enable selector-no-qualifying-type*/
+
+/* stylelint-disable max-nesting-depth */
+.featured {
```
Comment (guilhermejcgois) [inline scss/_global.scss:139]: This we doesn't use, we can remove it

File: scss/_global.scss [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+    margin-top: 5px;
+  }
+
+  &.liquidity hr {
+    margin-top: 10px;
+  }
+}
+/* stylelint-enable max-nesting-depth */
+
+.featured-text {
```
Comment (guilhermejcgois) [inline scss/_global.scss:173]: same here as above

**Pull request merged**